pyomo/contrib/mindtpy/cut_generation.py

```diff
@@ -146,6 +170,19 @@ def add_oa_cuts(target_model, dual_values, solve_data, config,
 
 
 def add_int_cut(var_values, solve_data, config, feasible=False):
+    """
+    Modifies the model by adding integer cuts (if needed)
+
+    Parameters
+    ----------
+    var_values: TODO: Fill this in
+        values of all the fixed variables
+    solve_data: MindtPy Data Container
+        data container that holds solve-instance data
+    config: ConfigBlock
+        contains the specific configurations for the algorithm
+    feasible: bool, optional TODO: Should this be removed? It is not used
```

Comment:
feasible will probably be useful for GBD

Reply:
Not only that, maybe we want to keep track of those solutions that have a corresponding feasible or infeasible NLP and add their integer cuts in different "buckets". Let's leave this here.
The description should read: boolean indicating if integer combination yields a feasible or infeasible NLP.